README.md

```diff
@@ -869,6 +869,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 [[database](https://crates.io/keywords/database)]
 
 * [sfackler/r2d2](https://github.com/sfackler/r2d2) — generic connection pool [<img src="https://api.travis-ci.org/sfackler/r2d2.svg?branch=master">](https://travis-ci.org/sfackler/r2d2)
+* [launchbadge/sqlx](https://github.com/launchbadge/sqlx) - generic connection pool with async/await support and type enforcement [<img src="https://img.shields.io/github/workflow/status/launchbadge/sqlx/Rust?style=flat-square">](https://github.com/launchbadge/sqlx)
```

Comment:
The badge doesn't seem to specify a branch. Can you specifically focus on the master branch so we don't get build failures later on for other branches?

Reply:
Done! 👍

Reply:
I've just also realised that this should be in the "SQL" section just below. Can you move it down please?

Reply:
There you go. Since unlike the other items in the SQL category, it supports postgres/mysql/sqlite, I added a "generic" sub-category and put it there.

Reply:
Awesome, thanks